package.json

```diff
@@ -4,6 +4,7 @@
   "description": "A Karma plugin. Launcher for Chrome and Chrome Canary.",
   "main": "index.js",
   "scripts": {
+    "pretest": "standard",
```

Comment:
lets add a script called `lint` which runs standard so it's easy to run by itself